tests/unit/test_var_info.py

```diff
@@ -20,9 +20,8 @@ def setUpClass(cls):
         tests.
 
         """
-        cls.config_file = 'tests/unit/data/test_config.json'
+        cls.test_config_file = 'tests/unit/data/test_config.json'
         cls.namespace = 'namespace_string'
-        cls.sample_config = 'config/0.0.1/sample_config_0.0.1.json'
```

Comment:
It felt off that some of the tests were coupled to the sample configuration file from the `config` directory. I've added the minimal amount of information to the `test_config.json` file that is used everywhere else to make all the tests pass (primarily adding a couple of short name paths, and a couple of entries in the mapping between mission and short name). Now all the tests that use a configuration file use the one in the test directory, which feels better.